CMakeLists.txt

```diff
@@ -32,3 +32,8 @@ install(
     )
 
 install(FILES ${HEADERS} DESTINATION include)
+
+include_directories(src)
+
```

Comment:
Maybe add an option for testing? And/or do what's suggested here
https://cliutils.gitlab.io/modern-cmake/chapters/testing.html

Reply:
Should testing be built by default?

Reply:
Well that would be up to the maintainers I guess. However, personally I would have it on by default if it was the master project otherwise off. 
Because whenever I add dependencies via git submodules and add_subdirectory it's a bit annoying to have to disable everyone's tests and I silently thank those who do have that mechanism :)